test/e2e/utils_test.go

```diff
@@ -0,0 +1,151 @@
+package e2e_test
+
+import (
```

Comment:
This should all be moved into the e2e/helpers dir

Reply:
framework? I'll look into adding them when I get more consistent passing tests

Reply:
no i mean that the file you created utils_test.go should be in dir e2e/helpers/

Reply:
also moving the functions should be in the commit message as well (or seperate commit)

Reply:
I will mention in the commit :+1:

adding them in a separate package means we'd have to create public functions for the test/e2e/ package to use the helpers. seems a bit weird to only have shared code move to helpers, and have some left in mcd_test, yet defining an entirely separate package and moving all the helper code seems out of scope of this PR.

If I update the commit, would you be alright with me leaving them in utils_test for this PR?

Reply:
yeah leave it as is with updated commit, tests can be refactored later overall, i think. seems more important to get this all working right now. :)